src/main/java/com/uid2/admin/Main.java

```diff
@@ -231,7 +232,7 @@ public void run() {
             JobDispatcher jobDispatcher = new JobDispatcher("job-dispatcher", 1000 * 60, 3, clock);
             jobDispatcher.start();
 
-            val clientSideKeypairService = new ClientSideKeypairService(config, auth, writeLock, clientSideKeypairStoreWriter, clientSideKeypairProvider, siteProvider, keysetManager, keypairGenerator, clock);
+            ClientSideKeypairService clientSideKeypairService = new ClientSideKeypairService(config, auth, writeLock, clientSideKeypairStoreWriter, clientSideKeypairProvider, siteProvider, keysetManager, keypairGenerator, clock);
```

Comment:
Curious why you removed `var` here and `val` below?

Reply:
It was throwing errors on my side for some reason